tools/ngsderive.wdl

```diff
@@ -32,14 +55,19 @@ task infer_strandedness {
         ln -s ~{bam_index} "$CWD_BAM".bai
 
         ngsderive strandedness --verbose \
+            ~{if split_by_rg then "--split-by-rg" else ""} \
             -m ~{min_reads_per_gene} \
             -n ~{num_genes} \
             -q ~{min_mapq} \
-            -g ~{gtf} \
+            -g ~{gene_model} \
             "$CWD_BAM" \
             > ~{outfile_name}
 
-        awk 'NR > 1' ~{outfile_name} | cut -d$'\t' -f5 > strandedness.txt
+        if split_by_rg; then
+            awk 'NR > 1' ~{outfile_name} | cut -d$'\t' -f6 > strandedness.txt
+        else
+            awk 'NR > 1' ~{outfile_name} | cut -d$'\t' -f5 > strandedness.txt
```

Comment:
Do you need to specify `-d$'\t`? The tab delimiter should be the default for `cut`.

Reply:
Good point. We can _cut_ that extra argument

Reply:
Ah. Also just now realizing this won't work for the first case no matter what. We're going to need to add a `grep 'overall'` _after_ v4 of ngsderive is released (in progress [here](https://github.com/stjudecloud/ngsderive/pull/139)). So this will just be broken till that release. There's no sensical value to put in `strandedness.txt` with the current latest release of ngsderive when `--split-by-rg` is true. I think that's fine for now. We aren't using the problem configuration in prod.

I will make a note of the broken state though.